wp-job-manager-functions.php

```diff
@@ -74,8 +74,12 @@ function get_job_listings( $args = [] ) {
 		}
 
 		if ( ! empty( $args['search_location'] ) ) {
-			$location_meta_keys = [ 'geolocation_formatted_address', '_job_location', 'geolocation_state_long' ];
+			if ( __( 'remote', 'wp-job-manager' ) === $args['search_location'] || __( 'Remote', 'wp-job-manager' ) === $args['search_location'] ) {
```

Comment:
We could probably convert to lowercase before comparing. Less things to translate and handles all the case combinations.

```suggestion
			if ( strtolower( __( 'remote', 'wp-job-manager' ) ) === strtolower( $args['search_location'] ) ) {
```

<img width="618" alt="Screen Shot 2022-06-13 at 14 18 51" src="https://user-images.githubusercontent.com/2578542/173332916-674cead8-9883-46a2-9de7-d2f4104b463f.png">